docs/decisions/0006-versioning-of-event-transformers.rst

```diff
@@ -13,9 +13,13 @@ Event transformers may undergo modification in future in response to consumer re
 
 Decision
 --------
-#. Versions of event transformers will be maintained and emitted as part of  xAPI and Caliper events.
+#. Versions of event transformers will be maintained and emitted as part of xAPI and Caliper events.
 
-#. Transformer version will be defined by two non-negative integers X and Y representing major and minor version respectively in the form: X.Y.
+#. The "transformer version" will be a concatenation of the name of the transformer ("event-routing-backends"), an @ symbol, and the version of the event-routing-backends package used to generate the event.
```

Comment:
The idea is that different transformers can use this field so that LRSs which have inputs from different systems can identify both the source transformer and version and make downstream decisions on parsing based on that.